### PR TITLE
CIV-5171 - Hearing Notice API tests

### PR DESCRIPTION
**Pull request by m-meulendijks-v1**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-5171

### Change description ###
Added hearing notice api tests to SDO + DJ journeys

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: e2e/api/steps.js
Comment (m-meulendijks-v1) [inline e2e/api/steps.js:1037]: Tests are currently failing here, because the event end state is wrong. Once corrected, the tests should run as well.

Comment (m-meulendijks-v1) [reply e2e/api/steps.js:1047]: Fix has been merged

**Review by drummondjm — APPROVED** [review]

**Review by miguelMolina3691 — APPROVED** [review]

**Review by filippo-possenti-version1 — APPROVED** [review]

**Review by filippo-possenti-version1 — DISMISSED** [review]

**Review by drummondjm — DISMISSED** [review]

**Review by filippo-possenti-version1 — APPROVED** [review]

**Comment by filippo-possenti-version1** [comment]
Past approvals here got overwritten multiple times due to merges and rollback of a pending release. There's no need for further approvals. As soon as it turns green I'll merge this.

**Pull request merged**
